### ISSUE TITLE
Help please

### ISSUE THREAD

**Issue by jbarros35** [input-output-hk/offchain-metadata-tools#12]
I don't understand this "base 16" code in your tutorial. Can you please provide us more information? Right now its very hard to follow the instructions.

**Comment by sevanspowell** [comment]
Yes @jbarros35. It's very confusing, took me a while to figure out.

I wrote up my experience here, does this help you? https://input-output-hk.github.io/offchain-metadata-tools/#minting-your-first-asset

**Comment by sevanspowell** [comment]
Please re-open if you need more help, thanks!
